File: C++/05-Operadores Relacionais/Relacionais2.cpp
#include<iostream>
#include<stdlib.h>
#include<string>

using namespace std;

int main()
{
    //Podemos comparar tipos diferentes de variaveis
    //OBS: Variaveis de caracteres só podem ser comparadas com == ou !=
    int a = 5;
    float b = 5.0;
    char c = '5';
    bool teste;

    cout << "1- Verdadeiro / 0- Falso" << endl;

    teste = a == b;
    cout << a << " == [igual] " << b << " ? " << teste << endl; //verdadeiro(1)
    teste = a == c;
    cout << a << " == [igual] " << c << " ? " << teste << endl; //falso(0) c é caractere
    teste = a != c;
    cout << a << " != [diferente] " << c << " ? " << teste << endl; //verdadeiro(1)

    system("pause");

    return 0;
}

File: Python/05-Operadores Relacionais/Relacionais2.py
#Tambem podemos comparar caracteres usando os operadores relacionais

a = 'c'
c = 'c'
d = 6
e = '6'
nome1 = 'Cecilia'
nome2 = 'Maria'

print("{0} == [igual a] {1} ? {2}".format(a, c, a == c)) # Verdadeiro
print("{0} == [igual a] {1}[str] ? {2}".format(d, e, d == e)) # Falso pois um é string
print("{0} == [igual a] {1} ? {2}".format(nome1, nome2, nome1 == nome2)) # Falso
